src/components/Card.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+function Card() {
+    const [cardData, setCardData] = useState([]);
+    useEffect(() => {
+        axios.get('/api/products').then((data) => {
+            setCardData(data.data.products);
+        }).catch((err) => { console.error(err) });
```

Comment:
Suggestion:
For such api calls make a folder names `services` and make utility functions for these and import wherever you need